main.go

```diff
@@ -61,7 +59,25 @@ func initTracerProvider(ctx context.Context, logger *zap.SugaredLogger) func() {
 
 func main() {
 	logger := zap.Must(zap.NewProduction(zap.Fields(zap.String("service_name", "opg-sirius-workflow")))).Sugar()
-	apiCallLogger := logging.New(os.Stdout, "opg-sirius-workflow")
+	apiCallLogger := slog.New(slog.
+		NewJSONHandler(os.Stdout, &slog.HandlerOptions{
+			ReplaceAttr: func(groups []string, a slog.Attr) slog.Attr {
+				if a.Key == "level" {
+					return slog.Attr{}
+				}
+
+				if a.Key == "time" {
+					a.Key = "timestamp"
+				}
+
+				if a.Key == "msg" {
+					a.Key = "message"
+				}
+
+				return a
+			},
+		}).
+		WithAttrs([]slog.Attr{slog.String("service_name", "opg-sirius-workflow")}))
```

Comment:
If you import `github.com/ministryofjustice/opg-go-common/telemetry`, you should be able to simplify this to:
```suggestion
	apiCallLogger := telemetry.NewLogger("opg-sirius-workflow")
```